src/main/java/com/aws/greengrass/mqtt/bridge/clients/IoTCoreClient.java

```diff
@@ -207,10 +207,14 @@ private void subscribeToTopicsWithRetry(Set<String> topics) {
                 }, "subscribe-iotcore-topic", LOGGER);
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt();
+                return;
             } catch (Exception e) {
-                LOGGER.atError().kv(TOPIC, s).setCause(e).log("Failed to subscribe to IoTCore topic");
+                LOGGER.atError().kv(TOPIC, topic).setCause(e).log("Failed to subscribe to IoTCore topic");
             }
-        });
+        }
+
+        LOGGER.atTrace().kv("subscribedIotCoreTopics", String.join(",", subscribedIotCoreTopics))
+                .log("Subscription requests complete");
```

Comment:
We can skip `String.join()` here, not needed.
```
LOGGER.atTrace().kv("subscribedIotCoreTopics", subscribedIotCoreTopics).log("Subscription requests complete");
```

Reply:
thanks, fixed!